### DISCUSSION TITLE
Local only or local/remote selectable?

### DISCUSSION THREAD

**Discussion by MikeGuest** [tuya/tuya-home-assistant#98] [Q&A]
Hi.

Thanks for all the work on this integration - I honestly believe that it will be more likely I'll buy more tuya-hosted devices when this is fully released and working.

One thing is concerning me with this update (which in most cases is an improvement) - I look after tuya devices at 2 sites. Right now using the cloud-based integration I can control and manage devices at both sites. With local control, that will be restricted to devices on the same network.

Do you know if the plans are to allow both local and remote control to be selectable on a per device basis? I don't want to have to install a whole home automation setup at my elderly father in law's flat just to allow his automated curtains to work.

**Comment by tsutsuku** [comment]
@MikeGuest  We will consider it.
